setup.py

```diff
@@ -112,6 +112,7 @@
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
         "Programming Language :: Python :: 3.13",
+        "Programming Language :: Python :: 3.14",
```

Comment:
Note that ideally we release a new version of `pytest-reana` from https://github.com/reanahub/pytest-reana/pull/143 then bump the `pytest-reana` dependency here (in order to get a new pytest version range) and merge this one afterwards.